notes/v1.0.0-beta3.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Release binaries can be downloaded with the following links:
+
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_darwin_amd64.tar.gz>
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_linux_32-bit.deb>
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_linux_32-bit.rpm>
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_linux_386.tar.gz>
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_linux_64-bit.deb>
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_linux_64-bit.rpm>
+<https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_linux_amd64.tar.gz>
+
+Checksums can be found [here](https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_checksums.txt)
```

Comment:
```suggestion
Checksums  are  at [ecctl_1.0.0-beta3_checksums.txt](https://download.elastic.co/downloads/ecctl/1.0.0-beta3/ecctl_1.0.0-beta3_checksums.txt).
```

(Just about every style guide I know says to avoid "here" in links.)